packages/frontend/.cct/utils/get-template-component-webpack-config.js

```diff
@@ -1,28 +1,24 @@
 import { dirname, join } from "path";
 import webpack from "webpack";
 import MiniCssExtractPlugin from "mini-css-extract-plugin";
-import tailwindPostCssConfig from "../../tailwind.config.cjs";
 import { fileURLToPath } from "url";
 import { createRequire } from "module";
-import { long as longCommitHash } from "git-rev-sync";
```

Comment:
Wondering if we can remove the `git-rev-sync` dependency.

Reply:
It's "unfortunately" needed because it's still included in the end `base.json` file to be uploaded on IPFS. It's because each user with both the git repo URL and the commit hash can go and verify if the code is "right". To be honest though, we can probably remove it.

Reply:
Removed it in c1ce00ed37950aa4b7ce877b943f2d66c91ab900